build-utils/sentry-instrumentation.ts

```diff
@@ -11,58 +15,64 @@ const {
   GITHUB_SHA,
   GITHUB_REF,
 } = process.env;
+
 const IS_CI = !!GITHUB_SHA;
 
 const PLUGIN_NAME = 'SentryInstrumentation';
 const GB_BYTE = 1073741824;
 
-const createSignature = function (secret, payload) {
+const createSignature = function (secret: string, payload: string) {
   const hmac = crypto.createHmac('sha1', secret);
   return `sha1=${hmac.update(payload).digest('hex')}`;
 };
 
 class SentryInstrumentation {
+  initialBuild: boolean = false;
+
+  Sentry?: typeof Sentry;
```

Comment:
The alternative would be to type this as `any` 🤷‍♂️

Reply:
better` typeof Sentry `:D

Reply:
Yeah but now I have to have the node sdk in `dependencies`